src/Polly.Core/Simmy/Outcomes/OutcomeChaosStrategy.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Polly.Telemetry;
+
+namespace Polly.Simmy.Outcomes;
+
+#pragma warning disable S3928 // Custom ArgumentNullException message
+
+internal class OutcomeChaosStrategy<T> : MonkeyStrategy<T>
+{
+    private readonly ResilienceStrategyTelemetry _telemetry;
+
+    public OutcomeChaosStrategy(OutcomeStrategyOptions<Exception> options, ResilienceStrategyTelemetry telemetry)
+        : base(options)
+    {
+        if (options.Outcome is null && options.OutcomeGenerator is null)
+        {
+            throw new ArgumentNullException(nameof(options.Outcome), "Either Outcome or OutcomeGenerator is required.");
```

Comment:
We shouldn't really throw an ANE with the name of a parameter that doesn't exist.

Reply:
Any suggestions?

Reply:
Just throw `ArgumentException` or `InvalidOperationException`?